Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/Web.config

```diff
@@ -19,16 +19,15 @@
   </appSettings>
 
   <system.web>
-    <globalization culture="de-AT" uiCulture="en-US"/>
```

Comment:
I think that all of the `Web.config` changes are not needed and just caused by the tooling during a Nuget package upgrade. So we can reset all of them.